### ISSUE TITLE
CC a11y audit - Select component

### ISSUE THREAD

**Issue by amycole501** [uswds/uswds#5595]
Audit the select component for WCAG compliance and critical checklist checks using three manual accessibility testing methods.
[Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing)
This ticket will be considered done once the following content has been written and associated WCAG checks are in place for the select component component:
WCAG criteria related to [select component](https://designsystem.digital.gov/components/select/) elements
How to perform manual testing (critical checklists) and related user-focused questions for downstream teams to test. 
a. Zoom mag to 200% 
b. Keyboard 
c. JAWS 

### A11Y
- [ ] Establish the success criteria we need to check against 
- [ ] Perform each check 
- [ ] Review with other a11y team member
- [ ] Reassess (as needed)
- [ ] Submit any failures as bug tickets

### Content and Dev

- [ ] https://github.com/uswds/uswds-site/issues/2436

## References
[Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing)
[List of all components](https://docs.google.com/spreadsheets/d/1ASzZ80oI6WVWAifqPO6gYAqrsRaiZs_qGTOStuQPyKs/edit?usp=sharing)
